src/content/index.html

```diff
@@ -18,13 +19,13 @@
         <i class="ion-md-menu toggle_js"></i>
         <ul class="menu_js">
           <li>
-            <a href="x" >Home</a>
+            <a href="#search-bar" >Home</a>
           </li>
           <li>
-            <a href="x">Artists</a>
+            <a href="#root">Music</a>
           </li>
           <li>
-            <a href="x">Contact</a>
+            <a href="#footer_">Contact</a>
```

Comment:
nie pomylilas sie z tym znakiem: '_'?

Reply:
to nie blad :) dodalam nowe id bo nie dzialal odsylacz jesli dalam klase, ktora tam byla, zmienilam id na footer teraz